.pep8speaks.yml

```diff
@@ -2,7 +2,7 @@ scanner:
     linter: flake8
 
 flake8:
-    max-line-length: 120  # Default is 79 in PEP 8
+    max-line-length: 500  # Default is 79 in PEP 8
```

Comment:
Please revert to 120.

Reply:
For those long `return` statements, they can either be escaped or you cna add a
`#noqa: E501`
on to the end of the lines.

Reply:
Reverted in new commit.